docs/UserGuide.md

```diff
@@ -823,28 +847,23 @@ You should see Connectify display the command to be copied, as shown below. The
 
 **Unsuccessful Output**
 
-- If the specified company index is missing, you will get the following error message:
+- If the specified company/person index is missing or is not a positive integer, you will get the following error message:
 ```
 Invalid command format! 
 sharePerson: Shares instructions on how to add a Person, from the specified company, to another address book.
 Parameters: COMPANY_INDEX (must be a positive integer) PERSON_INDEX (must be a positive integer)
 ```
 
-- If the specified company index is invalid, you will get the following error message:
+- If the specified company index is invalid (larger than the index of the displayed company list), you will get the following error message:
```

Comment:
Great that you have separated the various unsuccessful output command result as such to make it clearer to the user what they should expect for incorrect outputs!